backend/app/views.py

```diff
@@ -243,3 +251,22 @@ def get_response_quiz_data(request, text_id):
         raise Http404 from text_not_exist
     res = get_quiz_questions(text_obj.content)
     return Response(res)
+def get_indiv_sentences(request, text_id):
+    """
+    API endpoint for getting the individual sentences from the given text.
+    """
+    text_obj = Text.objects.get(id=text_id)
+    sentences = get_sentences(text_obj.content)
+    # return Response(sentences)
```

Comment:
Looks like an errant comment left in here.

Reply:
The comment has also been removed